gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,7 @@
+#Mon Sep 04 14:35:28 WEST 2023
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-8.2.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.0-bin.zip
```

Comment:
I'm not super familiar with how Gradle versioning work, so this is just a question. We're moving (down?) from 8.2.1 to 8.0?

Reply:
Nice catch Hafiz, I didn't properly check the change, it was suggested by AS, and it said that the AGP version wasn't compatible, so I though it's updating it, but a downgrade is clearly not what I intended.
I'll double-check and update.